autorest/codegen/templates/templates_package_dataplane/CHANGELOG.md.jinja2

```diff
@@ -0,0 +1,6 @@
+# Release History
```

Comment:
this is an example of a file that we can share across package modes, it's not dataplane specific. Can you go through and reduce code duplication? It's very important for a codebase as large as autorest.python to not duplicate code